<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
border-image: url(:/Pics/Pics/Menu_BG.jpg);
}</string>
   </property>
   <widget class="QPushButton" name="pushButton_h264dec">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> qproperty-icon: url(:/Pics/Pics/Menu_button_Video.png);
qproperty-iconSize: 220px;
border:15px ;border-radius:15px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_peoplecnt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> qproperty-icon: url(:/Pics/Pics/Menu_button_Cam.png);
qproperty-iconSize: 220px;
border:15px;border-radius:15px

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2d_ghc">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> qproperty-icon: url(:/Pics/Pics/Menu_button_2D.png);
qproperty-iconSize: 220px;
border:15px;border-radius:15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sip_phone">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-icon: url(:/Pics/Pics/Menu_button_VoIP.png);
qproperty-iconSize: 220px;
border:15px;border-radius:15px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rtp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-icon: url(:/Pics/Pics/Menu_button_KWS.png);
qproperty-iconSize: 220px;
border:15px;border-radius:15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trust_demo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-icon: url(:/Pics/Pics/Menu_button_Security.png);
qproperty-iconSize: 220px;
border:15px;border-radius:15px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Logo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>-20</y>
      <width>461</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Pics/Pics/Nuvoton_Logo.png);
background:transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_2d_ghc</zorder>
   <zorder>pushButton_rtp</zorder>
   <zorder>pushButton_trust_demo</zorder>
   <zorder>pushButton_h264dec</zorder>
   <zorder>pushButton_peoplecnt</zorder>
   <zorder>pushButton_sip_phone</zorder>
   <zorder>label_Logo</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
